ams/ams-server/src/main/resources/sql/mysql/ams-init.sql

```diff
@@ -237,7 +237,6 @@ CREATE TABLE `optimize_task_history`
     `cost_time`       bigint(20) DEFAULT NULL COMMENT 'Task cost time',
     `queue_id`        int(11) DEFAULT NULL COMMENT 'Task queue id',
     `task_group_id`   varchar(40) DEFAULT NULL COMMENT 'Task group id',
-    PRIMARY KEY (`task_history_id`,`task_group_id`),
```

Comment:
Why remove this primary key?

Reply:
NOT NULL column is required for primary key, but `task_history_id` and `task_group_id` are DEFAULT NULL.